samples/snippets/activate_subordinate_ca.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START privateca_activate_subordinateca]
+import time
+
+import google.cloud.security.privateca_v1 as privateca_v1
+
+
+def activate_subordinate_ca(
+    project_id: str,
+    location: str,
+    ca_pool_name: str,
+    subordinate_ca_name: str,
+    pem_ca_certificate: str,
+    ca_name: str,
+) -> None:
+    """
+    Activate a subordinate Certificate Authority.
```

Comment:
```suggestion
    Activate a subordinate Certificate Authority (CA).
```

Reply:
This is just a writing suggestion -- it helps the reader to spell out the full phrase first and then indicate that you'll be using the acronym later.